.github/workflows/ci.yml

```
@@ -0,0 +1,42 @@
+name: Rails Tests and Lint
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
```

actionlint (1.7.9)

"services" section should not be empty. please remove this section if it's unnecessary

(syntax-check)